qt/python/mantidqt/mantidqt/widgets/sliceviewer/view.py

```diff
@@ -177,11 +177,7 @@ def create_axes_orthogonal(self, redraw_on_zoom=False):
         self.nonortho_transform = None
         self.ax = self.fig.add_subplot(111, projection='mantid')
         self.enable_zoom_on_mouse_scroll(redraw_on_zoom)
-        if self.grid_on:
-            self.ax.grid(self.grid_on)
-        if self.line_plots_active:
-            self.add_line_plots()
```

Comment:
I was just going to try and test locally but I seem to have a long compile ahead. Can I ask if the `self.add_line_plots()` was removed deliberately?

Reply:
Yes - it's explained in the second commit message. That function actually requires 2 arguments, so attempting to call it like this would throw an error anyway. We concluded it's a dead branch of code. And it looks like the intention was that it just ensures line plots stay on after changing the axes so in the worst case the user will just need to toggle them back on manually.

Reply:
Thanks. My apologies for not reading the commit messages. I agree it's dead and playing with the sliceviewer I can't obviously see the case that would benefit from it. I thought the dimension transposed caused the thing to clear the plots but they seem fine.